src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Char.cs

```diff
@@ -1478,11 +1490,15 @@ public static unsafe int IndexOfAny(ref char searchStart, char value0, char valu
                     goto NotFound;
                 }
 
-            Difference:
+            VectorMatch:
                 offset += (nuint)(uint)LocateFirstFoundChar(search);
+                goto Found;
+            }
+            else
```

Comment:
super nit: No need of `else{}`.

Reply:
Removed, also changed the last goto to `NotFound`